### PR TITLE
feat: remove pagination from examsOverview, load new exam with signalR

### PR DESCRIPTION
**Pull request by Djoleoo**

### CODE REVIEW

**Review by zelick — APPROVED** [review]
great

File: Konteh/back-office-app/src/app/app.module.ts [hunk 1]
```diff
@@ -62,6 +63,20 @@ import { environment } from '../environments/environment';
...
     },
+    {
+      provide: MAT_DATE_FORMATS,
+      useValue: {
+        parse: {
+          dateInput: 'LL',
+        },
+        display: {
+          dateInput: 'dd-MM-yyyy HH:mm:ss',
+          monthYearLabel: 'MMM YYYY',
```
Comment (MilosNesh) [inline Konteh/back-office-app/src/app/app.module.ts:74]: why do you have 3 Ms

File: Konteh/back-office-app/src/app/features/exams/exams-overview/exams-overview.component.html [hunk 2]
```diff
@@ -3,7 +3,8 @@ <h2>Exams Overview</h2>
 
   <mat-form-field class="search-input">
     <mat-label>Search by Candidate</mat-label>
-    <input matInput placeholder="Enter name or surname" [(ngModel)]="searchText" (ngModelChange)="onSearchChange($event)">
+    <input matInput placeholder="Enter name or surname" [(ngModel)]="searchText"
```
Comment (MilosNesh) [inline Konteh/back-office-app/src/app/features/exams/exams-overview/exams-overview.component.html:6]: Do you need both

File: Konteh/Konteh.BackOfficeApi/Features/Exams/GetExams.cs [hunk 3]
```diff
@@ -51,26 +43,18 @@ public Task<Response> Handle(Query request, CancellationToken cancellationToken)
 
             var exams = query
```
Comment (laslorajda) [inline Konteh/Konteh.BackOfficeApi/Features/Exams/GetExams.cs:44]: this should be awaited

File: Konteh/Konteh.BackOfficeApi/Features/Exams/GetExams.cs [hunk 4]
```diff
@@ -51,26 +43,18 @@ public Task<Response> Handle(Query request, CancellationToken cancellationToken)
...
             });
 
-            var response = new Response
-            {
-                Exams = examResult,
-                Length = length
-            };
-
-            return Task.FromResult(response);
+            return Task.FromResult(examResult);
```
Comment (laslorajda) [inline Konteh/Konteh.BackOfficeApi/Features/Exams/GetExams.cs:57]: Task.FromResult is an indication something should've been awaited but wasn;t

File: Konteh/Konteh.Infrastructure/Events/ExamRequestedEvent.cs [hunk 5]
```diff
@@ -3,5 +3,7 @@ public class ExamRequestedEvent
 {
     public string Name { get; set; } = string.Empty;
     public string Surname { get; set; } = string.Empty;
-    public string Email { get; set; } = string.Empty;
+    public int NumberOfQuestions { get; set; } = 0;
```
Comment (laslorajda) [inline Konteh/Konteh.Infrastructure/Events/ExamRequestedEvent.cs:6]: = 0 is implied

File: Konteh/back-office-app/src/app/features/exams/exams-overview/exams-overview.component.ts [hunk 6]
```diff
@@ -46,24 +46,26 @@ export class ExamsOverviewComponent implements OnInit {
...
-}
-
 
+  handleMessage(message: GetExamsResponse): void{
+    const foundExam = this.examList.find(exam => exam.id === message.id);
+    console.log("found exam: ", foundExam)
+    if (foundExam) {
+      foundExam.examStatus = message.examStatus;
+      foundExam.score = message.score;
+      this.dataSource.data = [...this.examList];
```
Comment (laslorajda) [inline Konteh/back-office-app/src/app/features/exams/exams-overview/exams-overview.component.ts:65]: do you need to do this?

**Pull request merged**